diff --git a/dist/index.js b/dist/index.js
index 63767167b9c492bb30edbe013285d8707a0bed40..28c92e363d6204d3a8c3abc20fcd332640ab3adc 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -46,6 +46,7 @@ export function expressDevServer(options) {
                     // explicitly typed since express handle function is not exported
                     const app = build.entry.module[exportName];
                     if (!app) {
+
                         return next(new Error(`Failed to find a named export "${exportName}" from ${entry}`));
                     }
                     // pass request to the Express app
